src/main/java/appeng/parts/misc/PartStorageBus.java

```diff
@@ -402,6 +403,21 @@ private void resetCache()
 
 	}
 
+	private int createHandlerHash( TileEntity target )
+	{
+		final EnumFacing targetSide = this.getSide().getFacing().getOpposite();
+		if( target.hasCapability( Capabilities.STORAGE_MONITORABLE_ACCESSOR, targetSide ) )
+		{
+			return 0;
+		}
+		IItemHandler handler = target.getCapability( CapabilityItemHandler.ITEM_HANDLER_CAPABILITY, targetSide );
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "handler" which hides the field declared at line 115. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AHiddenFieldCheck)